plugin_interpreter/plugins/__harness_content/__harness_helper.py

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
I'm guessing this is just for local testing? The base class handles "Pending" and "Done" status updates automatically when the job is received by the plugin/output is sent from it.